### DISCUSSION TITLE
Apply model to different temperature?

### DISCUSSION THREAD

**Discussion by McAlexDudu** [deepmodeling/deepmd-kit#1568] [Q&A] [answered]
Hi~ I have question that I use VASP calculate AIMD in 600K, and dp train these data. I wonder if I could use the graph.pb to run LAMMPS in other temperature or just the same 600K? why?

**Comment by likefallwind** [comment, marked as answer]
Please see this:   https://github.com/deepmodeling/deepmd-kit/discussions/1316

If you let dp learn the Model in 600K, your model will only learn situation in 600K(or around 600K). Other temperatures, especially have big difference in 600K, you should train dp again

**Reply by McAlexDudu** [reply]
Great，thanks!
